content/docs/cms.md

```diff
@@ -24,31 +23,31 @@ const cms = new TinaCMS()
 
 The `TinaCMS` constructor receives an object that can be used to configure CMS behavior. See [CMS Configuration](#cms-configuration) for details.
 
-### The &lt;Tina&gt; Component
+### The <TinaProvider> Component
```

Comment:
this renders as shown:

![image](https://user-images.githubusercontent.com/15221702/76535355-1a06c000-6451-11ea-8bbf-6b9f07e3c772.png)

Reply:
```suggestion
### The &lt;TinaProvider&gt; Component
```

Reply:
Is this an API change (hence the docs changes)?